gcp/gcp_setup.sh

```diff
@@ -0,0 +1,29 @@
+
+git clone https://github.com/facebookresearch/llama.git
+git clone git@github.com:Andrew-Cain/llama-recipes.git --branch develop
+
+cd llama
+source download.sh 
+
+wget https://raw.githubusercontent.com/huggingface/transformers/main/src/transformers/models/llama/convert_llama_weights_to_hf.py
+
+python3 -m venv .venv  
+source .venv/bin/activate
+pip install git+https://github.com/huggingface/transformers
+pip install -e .
+pip install protobuf
+mkdir llama-2-7b/7B
+find llama-2-7b -maxdepth 1 -type f -exec mv {} llama-2-7b/7B/ \;
+cp tokenizer.model llama-2-7b/
+# pip install accelerate
+
+python convert_llama_weights_to_hf.py \
+    --input_dir llama-2-7b --model_size 7B --output_dir models_hf/7B
+
+# ssh with vscode
+
+export HUGGINGFACE_TOKEN=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
```

Comment:
Maybe don't leave your hf token like that...

Reply:
Thank you. Revoked now. Multiple large blunders were made :(